### ISSUE TITLE
Does PR in this repo count towards #hacktoberfest2020 ?

### ISSUE THREAD

**Issue by shivambhat** [AliceWonderland/hacktoberfest#7239]

**Comment by SoftCreatR** [comment]
Nope. See https://github.com/AliceWonderland/hacktoberfest/issues/5296

**Comment by vrbro** [comment]
Hey @shivambhat if you get to know any other way of checking (if PR counts in a repo) except:-

```
_

1. > Asking the repo owner in an issue
2. > Checking hacktoberfest label

_
```
Then please tell

**Comment by shivambhat** [comment]
Till now honestly I checked few repos where I checked the label hacktoberfest2020 and contributed 8-9 times in few repos and made PR's out of which only 2/9 got approved and currently under review.
But to take PR's u need to propose changes in documentation or code and if the maintainer finds it useful, they will merge it within 2-3 days and if that takes more time, then consider it ineligible for hacktoberfest2020
But I will get back to you & confirm with my friend.

**Comment by shivambhat** [comment]
Once you submit a PR  in any repo, you will be able to see in dashboard if it is eligible for  hacktoberfest2020 or not?

Dashboard link:https://hacktoberfest.digitalocean.com/profile

**Comment by DishantIsrani** [comment]
hi @shivambhat can you guide me how to do this i have been doing this for the first time and learning yet can you help?

**Comment by keshavsingh3197** [comment]
> hi @shivambhat can you guide me how to do this i have been doing this for the first time and learning yet can you help?

@DishantIsrani don't be panic if your are doing this first time . If you stuck in any problem then search google or find the solution of that problem on youtube etc... **solution is avilable of each problem mostly online.**

If you are capable of finding solution yourself then it is better for you.

**Comment by sauravrishu9810** [comment]
I would like to contribute as I posses all the skills required to fulfill the project
